.github/workflows/_build-wheel-release-upload.yml

```diff
@@ -12,9 +12,40 @@ on:
       PYPI_TOKEN:
         description: 'PyPI token'
         required: true
+      PAT_TOKEN:
```

Comment:
GitHub workflows themselves have no information about the admin privileges of the one who started this workflow.

Therefore, we must provide a `PAT` (Personal Access Token) for this workflow so that it can use the privileges of `sbillinge` i